pandas/core/groupby/groupby.py

```diff
@@ -734,7 +734,7 @@ def pipe(self, func, *args, **kwargs):
 
     plot = property(GroupByPlot)
 
-    def _make_wrapper(self, name):
+    def _make_wrapper(self, name: str) -> Callable:
```

Comment:
add type parameters for Callable (if you can)

Reply:
its going to end up being `wrapper(self, *args, **kwargs) -> FrameOrSeriesUnion`.  Is that `Callable[..., FrameOrSeriesUnion]`?

Reply:
sure